crates/release_plz_core/src/pr.rs

```diff
@@ -135,29 +144,29 @@ fn pr_body_default(
     packages_to_update: &PackagesUpdate,
     project_contains_multiple_pub_packages: bool,
 ) -> String {
-    let header = "## 🤖 New release";
-
-    let summary = packages_to_update.summary();
-    let changes = {
-        let changes = packages_to_update.changes(project_contains_multiple_pub_packages);
-        format!(
-            "<details><summary><i><b>Changelog</b></i></summary><p>\n\n{changes}\n</p></details>\n"
-        )
-    };
+    let mut context = tera::Context::new();
+    context.insert("summary", &packages_to_update.summary());
+    context.insert(
+        "changes",
+        &packages_to_update.changes(project_contains_multiple_pub_packages),
+    );
```

Comment:
I saw you added other variables to the context (summary and changes).
Instead, it would be nice to re-use the same variables we already documented in https://release-plz.ieni.dev/docs/config#the-pr_body-field